Makefile

```diff
@@ -5,6 +5,7 @@ export GOROOT=/usr/local/go
 export GOPATH=$(HOME)/go
 export GOBIN=$(GOPATH)/bin
 export GO111MODULE=on
+export GOFLAGS=-modcacherw
```

Comment:
does this compile for 201911/202012 branch?

Reply:
This option is available starting go version `1.14.x`.

I believe master & 202012 uses 1.14.x go version. 

Not sure about 201911. 

@sumukhatv, can you provide more  info about the versions the other images use?

Reply:
we use go `1.14` in 201911: https://github.com/Azure/sonic-restapi/blob/8d09602a9e3c762f86992d8eaf642e7115c5318b/go-server-server/go.mod#L3